### ISSUE TITLE
Ctrl + Enter not running code

### ISSUE THREAD

**Issue by rjake** [dbeaver/dbeaver#16077]
#### System information: 
- Operating system (distribution) and version: Windows 10 Enterprise
- DBeaver version: 22.0.2
- Additional extensions: none

#### Connection specification:
- Database name and version: Netezza 7.2.0.6
- Driver name: JDBC
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:
When I run ctrl + enter on a script with multiple statements (separated by `;`) or hit the execute SQL statement button ▶️ it just runs `;` and shows: 
![image](https://user-images.githubusercontent.com/14003618/161618136-3e4bae68-5628-4440-b197-1c84991f5a70.png)

This started happening with the latest upgrade from 22.0.1 to 22.0.2 today. My settings have not changed and show:
![image](https://user-images.githubusercontent.com/14003618/161618409-0d4bf15b-c1fb-4193-96bf-82ce027fc864.png)

#### Steps to reproduce:
These only run correctly if I highlight the text for the query I want to run before hitting ctrl + enter or the run SQL button ▶️  
```sql
select * from _v_table
;

select * from _v_session
;
```

**Comment by ShadelessFox** [comment]
Hello @rjake,

Thanks for the bug report.

**Comment by marktsug** [comment]
I'm experiencing a similar issue running Version 22.0.2.202204021640 on Cloudera Impala.  I can't run multi-line SQL commands in the SQL Editor.  If I put all the commands on a single line, it runs fine, but if there are multiple lines, the SQL command does not run and I get the same response that @rjake described above.  Help!

**Comment by serge-rider** [comment]
It happens when you have a semicolon on a separate line after a query. E.g.
```sql
select * from tableA
;
```
Workaround is to put semicolon in the end of query.

This bug was already fixed. Hotfix patch was applied to version 22.0.2, you can re-download it from dbeaver.io website or from GitHub.

**Comment by uslss** [comment]
verified

**Comment by rjake** [comment]
@serge-rider @ShadelessFox I am still having this issue. I tried installing from both
https://dbeaver.io/files/ea/  (Early access)
and
https://dbeaver.io/download/ (Prod)

I see version `22.0.2.2022 04 04 1542` (windows 64) and Check for Updates says there are no new versions. Is there another place I should download this from or maybe the 🐛 is still here?

**Comment by joshcsmith13** [comment]
> @serge-rider @ShadelessFox I am still having this issue. I tried installing from both https://dbeaver.io/files/ea/ (Early access) and https://dbeaver.io/download/ (Prod)
> 
> I see version `22.0.2.2022 04 04 1542` (windows 64) and Check for Updates says there are no new versions. Is there another place I should download this from or maybe the 🐛 is still here?

same here.  I'm not getting a new version and the bug still exists.

**Comment by ShadelessFox** [comment]
Hello @rjake @joshcsmith13,

Can you please provide a sample of a query causing the bug to appear?

Additionaly, please post a screenshot of the `About` dialog.

**Comment by techj82** [comment]
@ShadelessFox  I have the same issue as @rjake @joshcsmith13. Same code example as in previous posts apply. If the delimiter is on a new line it doesn't work. To be clear I get the same response as the original poster.

**This doesn't work**
```
select * from tableA
;

select *
from tableA
where blah
and blah
;
```

**This does work**
```
select * from tableA;

select * 
from tableB
where blah
and blah;
```

**Comment by joshcsmith13** [comment]
2 lines is all it takes to reproduce this.  the only way to get this to run is to highlight the select line, without the ";", and execute.

![image](https://user-images.githubusercontent.com/8902423/162475016-29bd1e1c-cbb6-4f89-8324-b579d66341d2.png)

I am connected to Netezza.
DBeaver Version 22.0.2.202204041542

![image](https://user-images.githubusercontent.com/8902423/162474415-3420494f-dc9c-4393-8ecb-92c3ec2ef92b.png)

I don't see any new packages since 2022-04-04, in Prod or EA.  When was this hotfix posted?  Shouldn't it have a new version number?

**Comment by joshcsmith13** [comment]
Is this because we're running a version from the future??  😄
![image](https://user-images.githubusercontent.com/8902423/162475642-7632b6cd-d903-4c36-a141-829b5d233ac1.png)

**Comment by rjake** [comment]
My details are the same as @joshcsmith13

**Comment by joshcsmith13** [comment]
I see a new [duplicate] ticket attached to this one, but nothing new.
Has anybody heard anything?

I suppose it might be worth noting that this bug was not present when querying Snowflake.  (That surprised me - that it was DBMS specific.)

**Comment by mradeschnig** [comment]
Exactly. On mysql and postgresql I have no problems. On SAP HANA I have the problems (https://github.com/dbeaver/dbeaver/issues/16135). I use the latest version 22.0.2.

**Comment by joshcsmith13** [comment]
I'm hoping that latest status update from @dbeaver-pm  to remove the "wait for response" label means that somebody from the team will be looking into this.  It is confusing that we were originally told there was a hotfix, but nobody has had any luck with it.

**Comment by LonwoLonwo** [comment]
Please check the "Script delimiter redefiner" connection setting in the "Advanced parameters".
Set it to empty if it's not.
I hope it helps.

![2022-04-15 12_21_54-Window](https://user-images.githubusercontent.com/45152336/163553341-658f1395-9d7a-40a4-bc4f-08724c559e99.png)

**Comment by LonwoLonwo** [comment]
The New DBeaver version is available. 
https://dbeaver.io/download/
Could you please download the latest version and see if the issue is gone there?

**Comment by marktsug** [comment]
> Please check the "Script delimiter redefiner" connection setting in the "Advanced parameters". Set it to empty if it's not. I hope it helps.
> 
> ![2022-04-15 12_21_54-Window](https://user-images.githubusercontent.com/45152336/163553341-658f1395-9d7a-40a4-bc4f-08724c559e99.png)

this solution worked for me using Cloudera Impala JDBC Driver 2.5.41.  thanks.

**Comment by ceinmart** [comment]
Just downloaded the version 22.0.3 and the problem still occurring with informix connection. 
The solution mentioned above , about set delimiter redefiner on driver settings works for me too. 
I have a ton of connections, but if set from "Database -> Driver Manager" , you set the configuration to all connection. 
That was a acceptable solution.... 
annoying...but acceptable.

**Comment by LonwoLonwo** [comment]
We are in the fixing process here https://github.com/dbeaver/dbeaver/issues/16212

**Comment by Matvey16** [comment]
This is caused by previous default values in mentioned above script delimeter redifiner. Should be fixed in 22.0.4 with [#16218](https://github.com/dbeaver/dbeaver/pull/16218)

**Comment by joshcsmith13** [comment]
Yay - it was fixed when I downloaded 22.0.4.202205011839!

It is a little weird that if I leave my work-around extra blank line before the semicolon, and then put my cursor in front of the semicolon and execute, that it executes the next statement!?

![image](https://user-images.githubusercontent.com/8902423/166292254-638676fb-72ac-4b0b-b5ce-80bc24618087.png)
